apps/etterlatte-behandling/src/main/kotlin/behandling/domain/SamsvarMellomPdlOgGrunnlag.kt

```diff
@@ -50,4 +50,9 @@ sealed class SamsvarMellomPdlOgGrunnlag {
     data class Grunnbeloep(
         override val samsvar: Boolean
     ) : SamsvarMellomPdlOgGrunnlag()
+
+    @JsonTypeName("INSTITUSJONSOPPHOLD")
+    data class INSTITUSJONSOPPHOLD(
+        override val samsvar: Boolean
```

Comment:
Er det bare meg eller er det rart å ha samsvar for ting som ikke har med PDL og gjøre? Dette gjelder også Grunnbeløp typen.

Reply:
Det handler vel om at det ikke er samsvar mellom kilden vi lytter på og Doffen? Det burde kanskje vært fristilt fra "PDL".

Reply:
Ja det er min oppfatning, med mindre noen mener at grunnlagsendringshendelser kun kan være fra PDL.